<launch>
  <!-- <include file="/etc/ros/fuerte/stacks/openni_launch/openni.launch"/>  -->
  <node name="data_collection_code" pkg="tabletop_pushing" type="save_kinect_data" respawn="false" output="screen">
    <remap from="color_image_topic" to="/camera/rgb/image_rect_color"/>
    <remap from="depth_image_topic" to="/camera/depth_registered/image_rect"/>
    <remap from="point_cloud_topic" to="/camera/depth_registered/points"/>

    <param name="display_wait_ms" value="3" type="int"/>
    <param name="workspace_frame" value="/camera_link" type="string"/>
    <param name="img_output_path" value="/home/thermans/Dropbox/Data/aware_home_saliency/realtime" type="string"/>
    <param name="cam_info_topic" value="/camera/rgb/camera_info" type="string"/>
    <param name="save_all" value="false" type="bool"/>

    <param name="min_workspace_x" value="-2.0" type="double"/>
    <param name="max_workspace_x" value="2.75" type="double"/>
    <param name="min_workspace_z" value="-2.0" type="double"/>
    <param name="max_workspace_z" value="2.0" type="double"/>
    <param name="min_table_z" value="-2.0" type="double"/>
    <param name="max_table_z" value="2.0" type="double"/>

    <param name="pcl_cluster_tolerance" value="0.03" type="double"/>
    <param name="pcl_min_cluster_size" value="50" type="int"/>
    <param name="pcl_max_cluster_size" value="25000" type="int"/>
    <param name="pcl_voxel_downsample_res" value="0.005" type="double"/>
    <param name="pcl_cloud_intersect_thresh" value="0.0025" type="double"/>
    <param name="pcl_concave_hull_alpha" value="0.1" type="double"/>
    <param name="use_pcl_voxel_downsample" value="false" type="bool"/>
    <param name="table_ransac_thresh" value="0.015" type="double"/>
    <param name="table_ransac_angle_thresh" value="5.0" type="double"/>

  </node>
</launch>
